models/marts/core/int_all_graph_resources.sql

```diff
@@ -29,8 +29,8 @@ unioned_with_calc as (
             when resource_type = 'source' then null
             else {{ dbt.split_part('name', "'_'", 1) }}||'_' 
         end as prefix,
-        {{ dbt.replace("file_path", "regexp_replace(file_path,'.*/','')", "''") }} as directory_path,
-        regexp_replace(file_path,'.*/','') as file_name 
+        replace(file_path,regexp_replace(file_path,'.*{{ get_regexp_directory_pattern() }}', ''),'') as directory_path,
```

Comment:
I think @b-per will be able to explain this better than me

Reply:
@joellabes
after discussion during our bug bash we agreed that we need to include dbt.replace to accommodate adapters properly. We had originally removed it because we couldn't include the `{{ get_regexp_directory_pattern() }}` macro since there wasn't a good way of properly escaping quotation marks on `{{ dbt.replace("file_path", "regexp_replace(file_path,'.*/','')", "''") }}` when added.

I instead created an additional macro that will output this whole line of code properly for either Mac/Win and it works now:
`{{ dbt.replace("file_path", "regexp_replace(file_path,'.*/','')", "''") }}`